tests/system/src/testscriptapp1/java/mymod/pseudoScriptEngine/RgfPseudoScriptEngineFactory.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package pseudoScriptEngine;
+
+import javax.script.ScriptEngine;
+import javax.script.ScriptEngineFactory;
+
+import java.lang.StringBuilder;
+import java.util.Arrays;
+import java.util.List;
+
+
+public class RgfPseudoScriptEngineFactory implements ScriptEngineFactory
+{
+    final static String       ENGINE_NAME       = "RgfPseudoScriptLanguage (RPSL) 1.0.0";
+    final static String       SHORT_ENGINE_NAME = "rpsl";
+    final static String       ENGINE_VERSION    = "100.20200220";
+    final static List<String> EXTENSIONS        = Arrays.asList("rpsl", "RPSL" );
+    final static String       LANGUAGE_NAME     = "RgfPseudoScriptLanguage";
+    final static String       LANGUAGE_VERSION  = "1.0.0.100.20200220";
+
+    final static List<String> MIME_TYPES        = Arrays.asList("text/rpsl", "application/x-rpsl");
+    final static String       THREADING         = "MULTITHREADED";
+
+    final static List<String> ENGINE_NAMES = Arrays.asList(SHORT_ENGINE_NAME, "RgfPseudoSL");
```

Comment:
Here is an example where I think it is fine to line them up. Can you also do that for the last of these?

Also, our general pattern is `static final` (static goes first). I probably missed noting that in other places.